src/site/stages/build/drupal/metalsmith-drupal.js

```diff
@@ -92,6 +92,11 @@ function createHealthCareRegionListPages(page, drupalPagePath, files) {
       }
     }
   }
+
+  files[`drupal${drupalPagePath}/health-services/index.html`] = createFileObj(
+    page,
+    'health_care_region_health_services_page.drupal.liquid',
```

Comment:
At some point, we may just want to create map of `entityBundle`s to `layout`s that `createFileObj ` can reference instead of passing a second argument.